### ISSUE TITLE
[Issue]: Unable to install/start sd.next

### ISSUE THREAD

**Issue by Jaminanim** [vladmandic/automatic#3563]
### Issue Description

Attached is the log file as requested in issue #3492. I used ".\webui.ps1 --debug --use-cuda" (I have Nvidia) to start the program. Here is the log file:
[sdnext.log](https://github.com/user-attachments/files/17687147/sdnext.log)

I follow the instructions on the page [here](https://github.com/vladmandic/automatic). This issue happens whether I use PowerShell or CMD and whether I use Python 3.10.X or 3.11.X.

It gets to
"INFO     Version: app=sd.next updated=2024-11-02 hash=65ddc611 branch=master
                         url=https://github.com/vladmandic/automatic/tree/master ui=main"
and does not proceed.

Like mentioned in the above issue, I cannot use Ctrl+C to quit the command (whether I use PowerShell or CMD), and pressing it shows nothing in the terminal window.

My connection is stable and I have deleted sd.next and started again as vladmandic had suggested in issue #3492. I am also able to install other things with git and python/pip without issue.

Let me know if you need anything else from me. I have limited PC access today due to helping a friend move houses, but I _may_ be able to occasionally check in remotely with my PC.

### Version Platform Description

- Windows 11
- AMD Ryzen 3700X
- RTX 3090
- Tried with both Python 3.11.9 and 3.10.6
- I tried both PowerShell and CMD.
- Latest git pull.

### Relevant log output

```shell
Using VENV: D:\Image Generation\sd-next\venv
10:34:30-673288 INFO     Starting SD.Next
10:34:30-676287 INFO     Logger: file="D:\Image Generation\sd-next\sdnext.log" level=INFO size=1788 mode=append
10:34:30-677287 INFO     Python: version=3.11.9 platform=Windows bin="D:\Image
                         Generation\sd-next\venv\Scripts\Python.exe" venv="D:\Image Generation\sd-next\venv"
10:34:30-850820 INFO     Version: app=sd.next updated=2024-11-02 hash=65ddc611 branch=master
                         url=https://github.com/vladmandic/automatic/tree/master ui=main
```

### Backend

Diffusers

### UI

Standard

### Branch

Master

### Model

StableDiffusion 1.5

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by vladmandic** [comment]
thanks for providing the necessary info. 
seems like its hanging VERY early - first thing after printing version is detection of platform and python itself.
can you try to execute this:

```shell
venv\Scripts\activate
python

import platform
platform.platform(aliased = True, terse = False)
platform.release()
platform.machine()
platform.processor()
platform.system()
platform.python_version()
```

**Comment by Jaminanim** [comment]
No worries. Thank you.

Here is the output:

```shell
PS D:\Image Generation\sd-next> venv\Scripts\activate
(venv) PS D:\Image Generation\sd-next> python
Python 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.platform(aliased = True, terse = False)
'Windows-10-10.0.22631-SP0'
>>> platform.release()
'10'
>>> platform.machine()
'AMD64'
>>> platform.processor()
'AMD64 Family 23 Model 113 Stepping 0, AuthenticAMD'
>>> platform.system()
'Windows'
>>> platform.python_version()
'3.11.9'
```

**Comment by vladmandic** [comment]
and none of that gets printed - that makes no sense. what makes even less sense is that you cannot abort python even with ctrl+c.
and you've tried different python versions.

only thing i can thing of is to attempt to start sdnext using full python trace and see where it hangs.
note this is EXTREMELY verbose. once hang occurs, copy last few screens of text.

```shell
venv\Scripts\activate
python -m trace --trace launch.py --debug
```

**Comment by Jaminanim** [comment]
Ok, when I looked back through that trace I saw that the last subprocess it tried to run was:

```
installer.py(316): result = subprocess.run(f'"{git_cmd}" {arg}', check=False, shell=True, env=os.environ, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=folder or '.')
```

This, combined with it hanging at the:

```
Version: app=sd.next updated=2024-11-02 hash=65ddc611 branch=master
url=https://github.com/vladmandic/automatic/tree/master ui=main
```

indicated to me that it may have been a problem with Git. So I reinstalled Git to 2.47 (from 2.45), and sd.next is now installed and working. It's strange that automatic and forge didn't see this issue, but it is fixed nonetheless and if anyone Googles the same issue hopefully they find this and it works for them.

Thank you for helping me, I really appreciate it!
